ebos/eclproblem.hh

```diff
@@ -2183,7 +2189,7 @@ private:
         initialFluidStates_ = tmpInitialFs;
         // make sure that the stuff which needs to be done at the beginning of an episode
         // is run.
-        this->beginEpisode(/*isOnRestart=*/true);
+        this->beginEpisode();
 
```

Comment:
why did you remove the (/*isOnRestart=*/true)?

Reply:
for some reason, it is not required and the `beginEpisode()` callback should not have such a parameter in the first place?!